src/features/courses/course-controller.ts

```diff
@@ -3362,6 +3363,75 @@ class CourseController {
             return result;
         });
     }
+
+    // Sends an email to the professor associated with the course.
+    async emailProfessor(options: EmailProfOptions): Promise<boolean> {
+        const coursePromise = Course.findOne({
+            where: {
+                id: options.courseId,
+            },
+            include: [{
+                model: User,
+                as: 'instructor',
+                attributes: ['id', 'firstName', 'lastName', 'email', 'preferredEmail'],
+                where: {
+                    active: true
+                },
+                required: true,
+            }],
+        });
+
+        const topicPromise = CourseTopicContent.findOne({
+            attributes: ['id', 'name'],
+            where: {},
+            include: [{
+                model: CourseWWTopicQuestion,
+                as: 'questions',
+                attributes: ['problemNumber'],
+                required: true,
+                where: {
+                    id: options.question.id,
+                    active: true,
+                }
+            }]
+        });
+
+        const [course, topic] = await Promise.all([coursePromise, topicPromise]);
+
+        if (_.isNil(course) || _.isNil(course.instructor)) {
+            throw new RederlyError('Could not find an instructor for this course.');
+        }
+
+        if (_.isNil(topic)) {
+            throw new RederlyError('Could not find a topic associated with the problem this student is trying to ask about.');
+        }
+        
+        const question = topic.questions?.[0];
+        
+        if (_.isNil(question)) {
+            throw new RederlyError('Could not find the question associated with the problem/topic this student is trying to ask about.');
+        }
+
+        const poorMansTemplate = `
+Hello Professor ${course.instructor.lastName},
+
+    Your student ${options.student.firstName} ${options.student.lastName} is asking for help with
+Problem ${question.problemNumber+1} in the Topic ${topic.name}.
```

Comment:
Still not convinced on problemNumber +1, we will have to talk during scrum

Reply:
Whoops, you're right, I must've gotten mixed up, I thought problemNumber was 0-indexed.